fitbenchmarking/fitbenchmarking
# Add `__str__` for `FittingProblem` and `FittingResult`

#### Description of Work
This PR adds the `__str__` method for `FittingProblem` and `FittingResult` to allow easier debugging.

Fixes #949

#### Testing Instructions

1. Try printing a `FittingProblem` and `FittingResult` wrapped in `str` and check that the output is in a readable format. Also check if we want any more information in these classes to be printed.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
